frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -160,6 +182,10 @@ export const WorkspacePage = (): JSX.Element => {
               onClose={() => setHasSeenAnnouncement(true)}
               isOpen={isAnnouncementModalOpen}
             />
+            <EmergencyContactModal
+              onClose={() => setHasSeenEmergencyContact(true)}
+              isOpen={isEmergencyContactModalOpen}
+            />
```

Comment:
probably not the correct place to put this, since the modal will probably be instantiated in the admin header (when it is complete) (and it is also now when i realize that is why this feature is not in the app yet)

Add a TODO i guess, to move the auto-instantiation of the modal into the admin header when that is done?

Reply:
gotcha. will create this as a separate issue!

Reply:
Issue #4153